examples/adrv9009-iiostream.c

```diff
@@ -36,7 +36,7 @@
 #define MHZ(x) ((long long)(x*1000000.0 + .5))
 #define GHZ(x) ((long long)(x*1000000000.0 + .5))
 
-#define ASSERT(expr) { \
+#define iio_ASSERT(expr) { \
```

Comment:
It would be more consistent to use uppercase for all macros.

Reply:
done